ui/app/alert-config/new.tsx

```diff
@@ -81,6 +83,26 @@ function getSlackProps(
           }}
         />
       </div>
+      <div>
+        <p>Members</p>
+        <Label as='label' style={{ fontSize: 14 }}>
+          Slack usernames to ping for these alerts. If left empty, pings the
```

Comment:
change `ping` to `mention` so that there is no confusion about DM

Reply:
Also, can you test that it does not need member ID instead of username?
might require `<@memberID>` instead of `@userName`

Reply:
It's just string formatting @iamKunalGupta